packages/itwinui-react/src/styles.js/styles.module.css

```diff
@@ -2,5 +2,8 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-@import '@itwin/itwinui-variables';
-@import '@itwin/itwinui-css';
+
+@layer itwinui-v1, itwinui.v1, itwinui.v2, itwinui.v3;
```

Comment:
Why both are there: itwinui-v1 and itwinui.v1?

Reply:
`itwinui-v1` is currently used, i plan to change it to `itwinui.v1` in a future PR. we need to support both